src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -81,6 +88,10 @@ public void blacklistDatabases(String name) throws MaxwellInvalidFilterException
 		blacklistDatabases.add(compile(name));
 	}
 
+	public void excludeColumns(String name) throws MaxwellInvalidFilterException {
+		excludeColumns.add(name);
```

Comment:
in all the other patterns we do `compile(name)` which lets us do regexp-based exclusion.  should support the same pattern here.